app/screens/home/channel_list/categories_list/header/header.tsx

```diff
@@ -132,10 +142,15 @@ const ChannelListHeader = ({
             items += 1;
         }
 
+        if (canInvitePeople) {
+            items += 1;
+            separators += 1;
+        }
+
         bottomSheet({
             closeButtonId,
             renderContent,
-            snapPoints: [bottomSheetSnapPoint(items, ITEM_HEIGHT, insets.bottom), 10],
+            snapPoints: [bottomSheetSnapPoint(items, ITEM_HEIGHT, insets.bottom, separators, SEPARATOR_HEIGHT), 10],
```

Comment:
I do not see a reason why to add separators and separator height to the function for the snap points, I would just do something like this

```suggestion
            snapPoints: [bottomSheetSnapPoint(items, ITEM_HEIGHT, insets.bottom) + (separators * SEPARATOR_HEIGHT), 10],
```

Reply:
Done.